SettingsList.py

```diff
@@ -2667,36 +2668,77 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         },
     ),
     Combobox(
-        name           = 'shuffle_interior_entrances',
+        name           = 'interior_entrances',
         gui_text       = 'Shuffle Interior Entrances',
         default        = 'off',
         choices        = {
             'off':       'Off',
-            'simple':    'Simple Interiors',
-            'all':       'All Interiors',
+            'shuffle':   'Shuffle',
+            'mix':       'Mix',
+            'decouple_uni': 'Decouple (Unidirectional)',
+            'decouple_bi': 'Decouple (Bidirectional)',
+            'mix_decouple_uni': 'Mix/Decouple (Unidirectional)',
+            'mix_decouple_bi': 'Mix/Decouple (Bidirectional)',
         },
         gui_tooltip    = '''\
-            'Simple Interiors': 
-            Shuffle the pool of interior entrances which contains most Houses 
-            and all Great Fairies.
-    
-            'All Interiors':
-            Extended version of 'Simple Interiors' with some extra places:
-            Windmill, Link's House, Temple of Time and Kakariko Potion Shop.
-
-            When shuffling any interior entrances, trade quest timers are disabled 
+            The pool of interior entrances contains most Houses and
+            all Great Fairies, but excludes the entrances to Windmill,
+            Link's House, Temple of Time, and Kakariko Potion Shop.
+
+            'Shuffle': Shuffle interior entrances among each other.
+
+            'Mix': Shuffle interior entrances into the "Mixed" pool
+            together with any other entrance pools set to 'Mix'.
+
+            'Decouple': Shuffle interior entrances among each other,
```

Comment:
This isn't an available option in the choices list. I assume the intent was to remove this from the tooltip

Loving this branch, hope it gets merged!